odkx-src/xlsx-converter-reference.rst

```diff
@@ -329,12 +337,42 @@ The following prompt types are available in ODK-X Survey.
     - | Used to capture an audio recording.
   * - barcode
     - | Used to capture a barcode.
+  * - coptic_calendar_picker
+    - | Uses a date picker widget to capture Coptic dates (Non-Gregorian Calendar).
+  * - ethiopian_calendar_picker
+    - | Uses a date picker widget to capture Ethiopian dates (Non-Gregorian Calendar)
+  * - hebrew_calendar_picker  
+    - | Uses a date picker widget to capture Hebrew dates (Non-Gregorian Calendar)
+  * - islamic_calendar_picker 
+    - | Uses a date picker widget to capture Islamic dates (Non-Gregorian Calendar)
+  * - mayan_calendar_picker 
+    - | Uses a date picker widget to capture Mayan dates (Non-Gregorian Calendar)
+  * - nanakshahi_calendar_picker
+    - | Uses a date picker widget to capture Nanakshahi dates (Non-Gregorian Calendar)
+  * - nepali_calendar_picker  
+    - | Uses a date picker widget to capture Nepali dates (Non-Gregorian Calendar)
+  * - persian_calendar_picker 
+    - | Uses a date picker widget to capture Persian dates (Non-Gregorian Calendar)
+  * - taiwan_calendar_picker 
+    - | Uses a date picker widget to capture Taiwan dates (Non-Gregorian Calendar)
+  * - thai_calendar_picker
+    - | Uses a date picker widget to capture Thai dates (Non-Gregorian Calendar)
+  * - ummalqura_calendar_picker
+    - | Uses a date picker widget to capture Umm al-Qura dates (Non-Gregorian Calendar)
   * - date
     - | Uses a date picker widget to capture a date. Automatically adjusts for timezone.
+  * - date (non-gregorian)
```

Comment:
How is this different from the specific non-Gregorian calendars listed above? Not clear to me.